latest/src/app/documentation/demos/quick-list/quick-list.demo.html

```diff
@@ -140,9 +140,17 @@ <h3>Summary of Options</h3>
 
     <div id="code-examples">
       <h3 id="examples">Code &amp; Examples</h3>
+      <h6>Mandatory quick list</h6>
       <div class="clr-col-3">
-        <clr-quick-list [clrAllValues]="possibleOptions" [(clrValues)]="selectedOptions"
-                        [clrAddLabel]="getAddLabel()" [clrBlankOption]="BLANK_OPTION" [clrMandatory]="true">
+        <clr-quick-list  [clrAllValues]="possibleOptions"
```

Comment:
please also add "[(clrValues)]" to show a full blown example. you may leave it empty initially, but show it's content below the quicklist to verify the selected values.
See multilingual inputs as example -> below the component are the texts to verify them (heading: Output texts)